librz/analysis/cc.c

```diff
@@ -77,12 +77,12 @@ RZ_API char *rz_analysis_cc_get(RzAnalysis *analysis, const char *name) {
 	int i;
 	// get cc by name and print the expr
 	if (rz_str_cmp(sdb_const_get(DB, name, 0), "cc", -1)) {
-		eprintf("This is not a valid calling convention name\n");
+		RZ_LOG_ERROR("This is not a valid calling convention name\n");
 		return NULL;
 	}
 	const char *ret = sdb_const_get(DB, sdb_fmt("cc.%s.ret", name), 0);
 	if (!ret) {
-		eprintf("Cannot find return key\n");
+		RZ_LOG_ERROR("Cannot find return key\n");
```

Comment:
In both error messages it's better to note they relate to the calling convention and print the cc name itself too